pages/lend/index.tsx

```diff
@@ -171,49 +168,35 @@ const Lend: NextPage = () => {
   }, [walletPK]);
   //  ************* END FETCH USER BALANCE *************
 
-  //  ************* START CALC. USER DEPOSITS *************
-  // calculate user deposits
-  // async function calculateTotalUserDeposits(
-  //   marketReserveInfo: any,
-  //   honeyUser: any
-  // ) {
-  //   const totalUserDeposits = await calculateUserDeposits(
-  //     marketReserveInfo,
-  //     honeyUser
-  //   );
-  //   setUserTotalDeposits(Number(totalUserDeposits));
-  // }
-
-  // useEffect(() => {
-  //   if (marketReserveInfo && honeyUser)
-  //     calculateTotalUserDeposits(marketReserveInfo, honeyUser);
-  // });
-  //  ************* END CALC. USER DEPOSITS *************
-
   //  ************* START FETCH CURRENT SOL PRICE *************
   // fetches the current sol price
-  async function fetchSolValue(reserves: any, connection: any) {
-    const slPrice = await fetchSolPrice(reserves, connection);
+  async function fetchSolValue(reserves: TReserve, connection: Connection) {
```

Comment:
rename func as mentioned for borrow page

Reply:
done